tests/integration/test_application_variables.py

```diff
@@ -23,8 +23,8 @@ def test_google_analytics_code_and_credentials_are_present(self):
         self.get("/questionnaire/feedback/")
         self.assertStatusOK()
         self.assertInHead("gtm.start")
-        self.assertInHead(
-            'dataLayer = [{"form_type": "H", "survey_id": "0", "title": "Feedback test schema"}]'
+        self.assertRegexPage(
```

Comment:
Perhaps it would be clearer if loaded the actual tx_id from the response data json then did something like this here? 
```suggestion
        self.assertInHead(
            f'dataLayer = [{{"tx_id": "{actual["METADATA"]["tx_id"]}"}}, {{"form_type": "H", "survey_id": "0", "title": "Feedback test schema"}}]'
        )
```

Reply:
Added to all three, thanks!